### DISCUSSION TITLE
--no-name feature request

### DISCUSSION THREAD

**Discussion by habahabahaba** [eza-community/eza#1251] [Development]
Hello. I would like to request a --no-name option for a long view that would omit the file-name (or dir-name) from the long view. 

I know it sounds strange, but it could help in cases where one have to fit a lot of metadata in a limited horizontal  screen-space by calling eza several times with different options (think a couple of fields in a single row); in those cases having a file-name at the end of every row becomes redundant and just wastes valuable horizontal space.

**Comment by cafkafk** [comment]
> I know it sounds strange

Kinda lol. I think there is a proposal somewhere to just have columns specified by a string that decides what fields show up... an approach like that wold cover this. I do feel it's a bit too strange to have it's own flag.

**Comment by habahabahaba** [comment]
That proposal looks like a better approach overall, but my proposal might be easier to implement. In any case, thank you for your work.

**Comment by habahabahaba** [comment]
I want to add. If you decide to implement a string-based config, might I ask for a line-break symbol.
